Production/python/eostools.py

```diff
@@ -5,41 +5,38 @@
 import sys
 import os
 import re
-import pprint
 import shutil
 import io
 import zlib
-
-def setCAFPath():
-    """Hack to get the CAF scripts on the PYTHONPATH"""
-    caf = '/afs/cern.ch/cms/caf/python'
-
-    if caf not in sys.path:
-        sys.path.append(caf)
-setCAFPath()
-try:
-    import cmsIO
-except ImportError as e:
-    import logging
-    logger = logging.getLogger(__name__)
-    logger.warning(str(e))
-    logger.warning("wasn't able to import cmsIO, which this job might not need unless it uses EOS.")
+import subprocess
+
+def splitPFN(pfn):
+    """Split the PFN in to { <protocol>, <host>, <path>, <opaque> }"""
+    groups = re.match("^(\w\+)://([^/]+)/(/[^?]+)(\?.*)?", pfn)
```

Comment:
I think I missed one problem here, the backslash before the first + should not be there. Instead, it should be:
` groups = re.match("^(\w+)://([^/]+)/(/[^?]+)(\?.*)?", pfn)`
which for instance works for `root://eoscms.cern.ch//eos/cms/store/cmst3/user/jngadiub/ntuple` returning
`root`
`eoscms.cern.ch`
`/eos/cms/store/cmst3/user/jngadiub/ntuple`